lib/config/migrations/custom/gradle-lite-migration.ts

```diff
@@ -0,0 +1,18 @@
+import is from '@sindresorhus/is';
+import { mergeChildConfig } from '../../utils';
```

Comment:
this causes strict null check issues :-/

Reply:
To be honest I need help with this issue, I spend few hours and could not find workaround. Excluding `gradle-lite-migration.ts` did not help. ts-ignore comment as well

Reply:
we should probalby shift this migration to a later time, as it needs strict null checks for almost all code or new exclusions in `tsconfig.strict.json` (don't like that, as we need to add previously working files).

Reply:
Converted to draft

Reply:
should be unblocked now